### ISSUE TITLE
browser opens and closes immediately with no output

### ISSUE THREAD

**Issue by ckundo** [ckundo/auto-vo#9]
> When I run auto-vo --url https://smashingmagazine.com --limit 5 - a chrome page opens with smashingmagazine.
"Welcome to MacOS, Link, VoiceOver off".`
The chrome page closed automatically, and I'm back at the CLI with no output.

see https://github.com/ckundo/auto-vo/issues/6#issuecomment-868550395

**Comment by ckundo** [comment]
@shaal continuing discussion here. can you please update the package? 

`npm update -g auto-vo`

**Comment by shaal** [comment]
I ran that command, still having the same issue.
(btw, I'm using MacBook Pro, BigSur 11.4)

**Comment by ckundo** [comment]
@shaal please also check that VoiceOver Applescript is enabled as described here https://github.com/ckundo/auto-vo/issues/10#issuecomment-868641729

**Comment by shaal** [comment]
Yes, I already have that enabled
![image](https://user-images.githubusercontent.com/22901/123474467-ff197a00-d5c7-11eb-9e7a-57b59c497863.png)

**Comment by ckundo** [comment]
> another thing that might be missing is opening VoiceOver Utility and checking "Allow VoiceOver to be controlled with AppleScript".

**Comment by ckundo** [comment]
@shaal the screen shot you showed is from system preferences. the instructions I posted are from the VoiceOver Utility app.

**Comment by shaal** [comment]
@ckundo it worked! thank you!
